src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1556,3 +1556,13 @@ def progress_callback(acl_changes):
         failed_entries.append(acl_changes.batch_failures)
 
     namespace.progress_hook = progress_callback
+
+
+def get_not_none_validator(attribute_name):
```

Comment:
please add unit test for the validator. see https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_validators.py

Reply:
Done